content/blog/2021/cfp-extension.md

```diff
@@ -0,0 +1,16 @@
+Title: PyCon India Call for Proposals is being extended!
+Date: 2020-05-05 17:00
+Category: 2021
+Slug: cfp-extension
```

Comment:
Slug let's make it `cfp-2021-extension` instead?

Reply:
It will be `blog/2021/cfp-extension` anyway